CDCgov/prime-reportstream
# Bg 2447 setting clia for sender

This PR adds creates a property in HL7 configuration to reset the MSH-4.2 CLIA value depending on the sender id. This will allow all messages flowing to a STLT to have the same CLIA number for each individual sender. For example, all messages from SimpleReport can have a single CLIA. This was built to limit the volume of distinct CLIA arriving at a receiver.

Test Steps:
1. Run unit test "test cliaForSender" in HL7SerializerTests

1. Create a new receiver org and a CSV file with fake data
2. Set cliaForSender as in the translation property. EX:
     - translation:
          - cliaForSender:
               - FakeSender: 12D3456789
3. Create fake file that will be accepted by your new receiver. The Sender ID should be set to "FakeSender" and the testing lab CLIA should be something different that was set in the org.
4. Send in the file via CLI and ensure the CLIA in MSH-4.2 is set to the CLIA you specified in the org

## Changes
- added property to HL7Configuration class
- added code to HL7Serializer to test for a value in cliaForSender
- added unit test to HL7SerializerTests

## Checklist

### Testing
- [ ] Unit test "test cliaForSender"

## To Be Done
- #2447 
     - This issue must be merged and deployed before 2447 can be completed
